deploy/tests/conftest.py

```diff
@@ -83,3 +86,14 @@ def patch(self, task_name: str, wrap_task=False, wrap_config=False) -> firesim.T
 
     return TaskMocker(mocker)
 
+
+#References: https://stackoverflow.com/questions/44718670/mock-s3-decorating-pytest-fixture
+@pytest.fixture()
+def mock_s3_client():
+    # setup: start moto server
```

Comment:
This is another nitpicky thing.  The `mock_s3()` method isn't starting a server, is it?  I thought it's only swapping out the backend of boto to the moto mocks.  Starting a `moto_server` looks more like https://github.com/fsspec/s3fs/blob/18009c4ebdd19d688c3b10cd1dc7147dfaaae351/s3fs/tests/test_s3fs.py#L88

Reply:
Correct. I believe that the comment where I copied this code from is just worded poorly.